meshroom/ui/qml/WorkspaceView.qml

```diff
@@ -78,7 +78,7 @@ Item {
                 cameraInit: reconstruction ? reconstruction.cameraInit : null
                 tempCameraInit: reconstruction ? reconstruction.tempCameraInit : null
                 cameraInitIndex: reconstruction ? reconstruction.cameraInitIndex : -1
-                onRemoveImageRequest: reconstruction.removeAttribute(attribute)
+                onRemoveImageRequest: reconstruction.removeImage(attribute)
                 onAllViewpointsCleared: { reconstruction.removeAllImages(); reconstruction.selectedViewId = "-1" }
```

Comment:
We should now remove the call to removeAllImages here, as it creates an empty element in the undo stack.